src/io/flutter/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -76,32 +83,42 @@
   private final Map<String, TIntObjectHashMap<Pair<Integer, Integer>>> myScriptIdToLinesAndColumnsMap =
     new THashMap<>();
 
-  @Nullable private final String myDASExecutionContextId;
   private final boolean myRemoteDebug;
   private final int myTimeout;
   @Nullable private final VirtualFile myCurrentWorkingDirectory;
   @Nullable protected String myRemoteProjectRootUri;
 
+
+  private VmOpenSourceLocationListener myVmOpenSourceLocationListener;
+
+  @NotNull private final ObservatoryConnector myConnector;
+
+  private boolean remoteDebug = false;
```

Comment:
I'm going to assume we don't know if remote debugging makes sense for Flutter and that's why all the references to this exist. If we do know (or someone does and hasn't said) then there is some more cleanup to do.